src/casp/services/db/models.py

```diff
@@ -24,7 +24,7 @@ def __repr__(self):
 class CASModel(db.Base):
```

Comment:
Maybe outside the scope of this PR and something I'd also want @fedorgrab to weigh in on: These classes are used in several places in the project and aren't documented at all.  It's admittedly not that difficult to understand what they are, but someone coming to this fresh (or returning after a while) might be confused about at least some of the columns, like `model_file_path`.  I think this file would probably benefit from a quick documentation pass.

Reply:
Yeah, I agree 100%, we can add docstrings in rst format in class description. This will allow us to autogenerate docs for sphinx documentation page as well. Also, I think db should go out of services and be a part of data_manager boilerplate package in the repo.

Reply:
Agreed.  If that's ok, I might wait for a later PR here.  This one's already a bit chonky

Reply:
Totally, this could be like a separate PR itself.:)